src/components/LightSwitch.js

```diff
@@ -1,11 +1,31 @@
-import React from "react"
+import React, {useState} from "react"
 
-const LightSwitch = () => {
+const LightSwitch = (props) => {
+  
+  // const[lightSwitchStatus,setLightSwitchStatus] = useState(false)
+
+  //   const lightsOn = () => {
+  //     setLightSwitchStatus(lightSwitchStatus === false ? true:false)
+  //   }
+  //   let switchOnOff = lightSwitchStatus === true ? 'on': 'off'
+      // let bulbOn = lightSwitchStatus === true ? 'yellow': 'white'
+// changes the color of the lightbulb on the app based on the use state value of the light switch.
+      var bulbColor = () => {
+          if(props.lightSwitchStatus === 'off') {
+           return 'white'
+          } else if(props.lightSwitchStatus === 'on') {
+            return 'yellow'
+          }
+        }
```

Comment:
Check out the [conditional (ternary) operator](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Conditional_Operator) to make cleaner code for simple if / else if statements like this.